tfe/provider_next.go

```diff
@@ -0,0 +1,118 @@
+package tfe
```

Comment:
```suggestion
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0

package tfe
```

Reply:
BAH THANK YOU. Got one file, missed the other.